package.json

```diff
@@ -21,6 +21,8 @@
   },
   "devDependencies": {
     "babel-plugin-inline-react-svg": "^0.4.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "cz-conventional-changelog": "^2.0.0",
```

Comment:
anyway this is some benefit of conventional changelog https://github.com/semantic-release/semantic-release